src/main/java/DataBase/DataBaseController.java

```diff
@@ -70,19 +70,19 @@ public User getUser(String name) throws FileNotFoundException, ParseException {
             }
         }
         myReader.close();
-    // Return a default user if user doesn't exist
+        // Return a default user if user doesn't exist
         return new User("Default User","Password");
-
     }
-    /** Adds a list of wishlists data to a user's database
+    /**
+     * Adds a list of wishlists data to a user's database
+     *
      * @param listOfWishlists list of wishlists data to add to database
-     * @param user user of the data it belongs to
-     * @returns whether wishlist was successfully saved
-     * */
+     * @param user            user of the data it belongs to
+     */
     // JSONArray's library has errors, can ignore
     @SuppressWarnings("unchecked")
 
-    public boolean saveListOfWishlists(ListOfProductLists listOfWishlists, User user) throws IOException {
+    public void saveListOfWishlists(ListOfProductLists listOfWishlists, User user) throws IOException {
```

Comment:
for getlistofwishlist it the name of the user is a param. but for savelistofwishlists it takes the actual user object instead of the name. I was wondering why this is? would it be cleaner to keep it consistent?